packages/hledger/build.sh

```diff
@@ -0,0 +1,38 @@
+TERMUX_PKG_HOMEPAGE=https://hledger.org/
+TERMUX_PKG_DESCRIPTION="Robust, friendly, fast plain text accounting software"
+TERMUX_PKG_LICENSE="GPL-3.0-or-later"
+TERMUX_PKG_MAINTAINER="@erplsf"
+TERMUX_PKG_VERSION=1.43.2
+TERMUX_PKG_SRCURL=https://github.com/simonmichael/hledger/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=60b74c70ddfc6b84ca87debd2ac302aac754da3c0d9089821182e56796cb841e
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BUILD_DEPENDS="aosp-libs, libffi, libiconv, libgmp, zlib, ncurses"
```

Comment:
```suggestion
TERMUX_PKG_DEPENDS="libffi, libiconv, libgmp, zlib, ncurses"
TERMUX_PKG_BUILD_DEPENDS="aosp-libs"
```

Reply:
thanks, already done locally, will push in a sec.

Reply:
done.